src/apify_client/clients/resource_clients/schedule_collection.py

```diff
@@ -0,0 +1,57 @@
+from typing import Any, Dict, List, Optional
+
+from ..base.resource_collection_client import ResourceCollectionClient
+
+
+class ScheduleCollectionClient(ResourceCollectionClient):
+    """Sub-client for manipulating schedules."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the ScheduleCollectionClient with the passed arguments."""
+        super().__init__(*args, resource_path='schedules', **kwargs)
+
+    def list(self, *, limit: Optional[int] = None, offset: Optional[int] = None, desc: Optional[bool] = None) -> Dict:
+        """List the available schedules.
+
+        https://docs.apify.com/api/v2#/reference/schedules/schedules-collection/get-list-of-schedules
+
+        Args:
+            limit: How many schedules to retrieve
+            offset: What schedules to include as first when retrieving the list
+            desc: Whether to sort the schedules in descending order based on their modification date
+
+        Returns:
+            The list of available schedules matching the specified filters.
+        """
+        return self._list(limit=limit, offset=offset, desc=desc)
+
+    def create(
+        self,
+        *,
+        cron_expression: str,
+        is_enabled: bool,
+        is_exclusive: bool,
+        name: Optional[str] = None,
+        actions: List[Dict] = [],
+    ) -> Dict:
+        """Create a new schedule.
+
+        https://docs.apify.com/api/v2#/reference/schedules/schedules-collection/create-schedule
+
+        Args:
+            cron_expression: The cron expression used by this schedule
+            is_enabled: True if the schedule should be enabled
+            is_exclusive: True if the schedule is exclusive
```

Comment:
This would deserve a better explanation, like "Don't start scheduled actor or task if it is still running from the previous schedule".